.github/workflows/yaml-lint-checker.yaml

```diff
@@ -0,0 +1,68 @@
+---
+name: Yaml Lint Release Notes Pull Request
+on:
+  pull_request:
+    paths:
+      - releases/**/release-notes/**.yaml
+  # Allow manual triggering
+  workflow_dispatch: {}
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}
+  cancel-in-progress: true
+jobs:
+  yaml-lint-checker:
+    name: Release Notes Check Yaml 👀
+    runs-on: ubuntu-22.04
+    if: ${{ !github.event.pull_request.draft }}
+    steps:
+      - name: Cancel Previous Actions
+        uses: styfle/cancel-workflow-action@0.9.1
+        with:
+          access_token: ${{ github.token }}
+      - name: Check out code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: '0'
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.22.4' 
+      - name: Install go dependencies
+        run: go mod download
+      - name: Build go prog
+        run: go build -o validate_yaml .github/scripts/validate_yaml.go
+      - name: Validate YAML
+        id: validate_yaml
+        run: |
+          INVALID_FILES=""
+          # check diff for invalid yaml files
+          git diff --name-only ${{ github.event.pull_request.base.sha }} ${{ github.sha }} -- releases/ | grep -E '\.ya?ml$'
+          CHANGED_FILES=$(git diff --name-only $(git merge-base origin/master HEAD) -- releases/ | grep -E '\.ya?ml$' || true)
+          if [ -n "$CHANGED_FILES" ]; then
+          echo "Validating yaml files: \n${CHANGED_FILES}"
+          OUTPUT=$(./validate_yaml $CHANGED_FILES)
+            if [[ "$OUTPUT" == *"YAML Validation Failed"* ]]; then
+              # If there are invalid files, set the output variable and fail the job. GITHUB_ENV is a special file that 
+              # allows you to set env variables for subsequent steps in the same job
+              echo "invalid_files<<EOF" >> $GITHUB_ENV
+              echo "$OUTPUT" >> $GITHUB_ENV
+              echo "EOF" >> $GITHUB_ENV
```

Comment:
this works? not 100% sure that this works

Reply:
Verified on my local branch that this allows to create a multiline variable for use in the next step.  Will send a screenshot with echo later

Reply:
Release action from today's release meeting for reference : https://github.com/kubernetes-sigs/release-actions